Tweak.xm

```diff
@@ -269,10 +270,13 @@ static BOOL isLongHoldEnabled()
 - (void)scan
 {
 	//Only on 7 and later
-	if (kCFCoreFoundationVersionNumber >= CoreFoundationiOS7)
+	if (kCFCoreFoundationVersionNumber >= CoreFoundationiOS7 && kCFCoreFoundationVersionNumber < CoreFoundationiOS11)
 	{
 		[[objc_getClass("WFWiFiManager") sharedInstance] setValue:[NSNumber numberWithInt:-130] forKey:@"_rssiThreshold"];
-	}
+    }
+    else if (kCFCoreFoundationVersionNumber >= CoreFoundationiOS11) {
+        [[objc_getClass("WPScanRequest") alloc] setRssiThreshold:[NSNumber numberWithInt:-130]];
```

Comment:
Does this method actually affect the wifi scan initiated by the system or does this only change this instance of `WPScanRequest`? I would assume the latter